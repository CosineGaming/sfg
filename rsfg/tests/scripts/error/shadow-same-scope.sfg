fn main()
	var x = 5
	assert(x == 5)
	var x = 6
	assert(x == 6)

