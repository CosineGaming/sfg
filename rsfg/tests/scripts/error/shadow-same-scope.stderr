[ERROR] shadowing is illegal within the same scope with x at 4:6